platform/installation.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Platform Installation
+summary: Guidance on the different ways to install and use the platform
+reviewed: 2019-06-15
+---
+
+The Particular Service Platform can be used in different ways.
+
+- Monitoring production systems
+- Debugging during development
+- Presenting from within Visual Studio
+
+## Monitoring production systems
+
+The Platform Installers requires internet connectivity and performs several actions like verifying MSMQ and MSDTC installation. It is possible to [achieve the same result](/platform/installer/offline.md) without requiring internet connectivity.
+
+For usage on production environments it is possible and recommended to install the different components individually. Each component has individual installation instructions.
+
+### ServiceControl
```

Comment:
I think this is too much information to split up.

I would not link to the GitHub releases. In fact, for Monitoring the releases/latest doesn't even include compiled artifacts, only source zips. 

It would probably be best to only provide https://particular.net/downloads as a source of downloading any of this stuff, as it already takes the most recent version into account.

I really don't think it's necessary to do anything except say there's the 1) platform installer, 2) instructions for offline, and 3) individual installers on the downloads page.